pkg/cmd/image/list.go

```diff
@@ -181,10 +180,8 @@ type imagePrintable struct {
 	Platform string // nerdctl extension
 }
 
-func printImages(ctx context.Context, options types.ImageListCommandOptions, client *containerd.Client, imageList []images.Image) error {
+func printImages(ctx context.Context, imageList []images.Image, options types.ImageListCommandOptions, client *containerd.Client, w io.Writer) error {
```

Comment:
nit: can you move `client` to after `ctx` and before `imageList`? It's more consistent with other funcs where a `client` is used. (e.g., `(ctx, client, container/image/etc, option, io, etc)`)

Reply:
makes sense, updated.